exercise/templates/exercise/staff/inspect_submission.html

```diff
@@ -81,6 +81,13 @@
 						</form>
 					{% endif %}
 				{% endif %}
+				{% if not submission.force_exercise_points %}
+					{% if exercise.grading_mode == 1 %}
+						{% translate "GRADING_MODE_BEST_EXPLAIN" %}
+					{% elif exercise.grading_mode == 2 %}
+						{% translate "GRADING_MODE_LAST_EXPLAIN" %}
+					{% endif %}
```

Comment:
I don't think it's a good idea to hardcode enum values into the template, but I guess it's not possible to access the enum directly from the template. Could you generate the explanation string in the view code instead? Or maybe we could make a generic template tag that compares enum values by their string representation instead? Something like: `exercise.grading_mode|enum_equal:"BEST"`

Reply:
True. Now choosing the text on the view side and just passing the string

Reply:
Looks good!